accounts/migrations/0002_auto_20201022_1553.py

```diff
@@ -1,18 +0,0 @@
-# Generated by Django 3.1.1 on 2020-10-22 15:53
```

Comment:
After pulling in and merging the latest PR, I kept getting an error when trying to navigate to the create_event page, which was to do with the CustomUser model. StackOverflow suggested that I clear the existing migrations and then run the `makemigrations` and `migrate` commands again, which resolved the issue. How do I get the previous migrations back?

Reply:
When this happens what I would first do is just try to delete the `db.sqlite3` file. If you have an existing file that you have done work on and others it might happen that there are issues then with the migrations. By deleting the database this should give you a clean slate. Running `makemigrations` then should just show "No changes needed" and  on `migrate` it would just recreate the db with the most up-to-date models.

Reply:
To remove the the changes to the migrations you can run `git revert <commit_id>` and you will then have to push the changes to your fork (probably with `-f` at the end). You can get the commit ID from the list of commits above or by running `git log`.

Reply:
Cool, I'll keep that in mind going forward and delete the `db.sqlite3` file as the first port of call 👍 I ran into some trouble with reverting the branch though, so I had to do a hard reset instead and force push to the branch. A couple of my previous commits are now gone from the history, but all the migration files are reverted and my new changes are also added back in.